functions/csvLoader/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const ATTRIBUTES_COLLECTION = 'attributes';
+export const ATTRIBUTES_COLLECTION = 'featureAttributes';
```

Comment:
Maybe I would also add here FEATURE_ATTRIBUTES_COLLECTION_NAME.

Reply:
I used the same reasoning as in url-loader